mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/UserReindexingTask.java

```diff
@@ -37,27 +37,27 @@
     public static final TaskType USER_RE_INDEXING = TaskType.of("user-reindexing");
 
     public static class AdditionalInformation extends ReprocessingContextInformation {
-        private final User user;
+        private final Username username;
 
-        AdditionalInformation(User user, int successfullyReprocessedMailCount, int failedReprocessedMailCount, ReIndexingExecutionFailures failures, Instant timestamp) {
+        AdditionalInformation(Username username, int successfullyReprocessedMailCount, int failedReprocessedMailCount, ReIndexingExecutionFailures failures, Instant timestamp) {
             super(successfullyReprocessedMailCount, failedReprocessedMailCount, failures, timestamp);
-            this.user = user;
+            this.username = username;
         }
 
-        public String getUser() {
-            return user.asString();
+        public String getUsername() {
```

Comment:
It will change the json returned by the webadmin when asking the details.
I don't think it is a big deal a the moment, but we should be careful.
Do we have some tests failing on this ?